go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/ci-tools
 
-go 1.20
+go 1.21
+
+toolchain go1.21.3
```

Comment:
Could we do it without this line?

Reply:
no, tried.

Reply:
i manually removed the line as you suggested. lets see